playbooks/keyholders/NEW_SIGNER.md

```diff
@@ -0,0 +1,147 @@
+# New Keyholder Responsibilities
+
+This document outlines the responsibilities of a new root keyholder.
+Follow this document if you are a new holder!
+
+## Outline
+
+Congrats on becoming a new Sigstore Root Keyholder!
```

Comment:
also, should we add something here about what type of key we recommend purchasing?

Reply:
Adding a note about this in `OVERVIEW.md` for keyholder responsibilities